plugins/core/webview/src/q-ui/components/toolkitOptions.vue

```diff
@@ -82,9 +82,9 @@ export default defineComponent({
         feature(): Feature {
             return this.$store.state.feature
         },
-        existConnections(): AwsBearerTokenConnection[] {
-            return this.$store.state.existingConnections
-        }
+        // existConnections(): AwsBearerTokenConnection[] {
+        //     return this.$store.state.existingConnections
+        // }
```

Comment:
don't we still need this?

Reply:
Yes, we do, the next commit will revert this